data/imageset-config.yaml

```diff
@@ -10,17 +10,7 @@ mirror:
           - '4.6.13'
     graph: true
   operators:
-    - catalog: redhat-operators:v4.7
-    - catalog: certified-operators:v4.7
-      packages:
-        - name: couchbase-operator
-        - name: mongodb-operator
-          versions:
-            - '1.4.0'
-        - name: crunchy-postgresql-operator
-          channels:
-            - name: 'stable'
-    - catalog: community-operators:v4.7
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.8
```

Comment:
Let's keep this config as-is unless the config spec changes, since it's an example config for users.